### PR TITLE
Add sina plot

### PR DESCRIPTION
**Pull request by jr-nicola**
This PR adds the sina plots into the treeview.

At the moment, the treeview uses the trimmed tooltips, the sina plot uses the untrimmed ones. They use different methods to grab the clusterID, but will need to anyway due to structure of underlying ggplot. I've added the get_cluster_id() function back in for now, but this can be removed after the trimmed tooltips are fixed.

Since both will need to be updated after the trimmed tooltip PR in {tfpscanner} goes through, I'm okay with putting these through with different styles of tooltips (since tooltip fixes feels like a separate issue).

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -158,10 +170,19 @@ get_selected_cluster_id = function(widgetChoice,
   filename = get_filename(widgetChoice)
   g = readRDS(filename)
   built = suppressWarnings(ggplot2::ggplot_build(g))
-  n_layers = length(built$data)
-  ids = built$data[n_layers][[1]]["data_id"]
-  tooltips = built$data[n_layers][[1]]$tooltip
-  tooltip_ids = suppressWarnings(readr::parse_number(tooltips))
+  g = readRDS(filename)
+  built = suppressWarnings(ggplot2::ggplot_build(g))
```
Comment (russHyde) [inline R/utils.R:174]:
```suggestion
```

Remove the duplicated lines

**Review by russHyde — CHANGES_REQUESTED** [review]
This is good. There's just a slight issue, probably from a merge-fix, where two lines were duplicated

File: R/utils.R (hunk 1 above)
Comment (jr-nicola) [reply R/utils.R]: Thanks - I've removed that duplication :)

**Review by russHyde — APPROVED** [review]
LGTM

**Pull request merged**